Source/JavaScriptCore/b3/testb3_7.cpp

```diff
@@ -945,12 +978,13 @@ void testLICMControlDependentNotBackwardsDominant()
     generateLoopNotBackwardsDominant(
         proc, array,
         [&] (BasicBlock* loop, Value*) -> Value* {
+            auto arguments = cCallArgumentValues<unsigned *>(proc, loop);
```

Comment:
Can you remove space between `unsigned` and `*`? And do that for all the other places?

Reply:
Done!